### PR TITLE
Add `*.ir.md`

### PR DESCRIPTION
**Pull request by irancms**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowlegements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

We are a private web and email hosting and domain provider as a *.ir.md.
We provide each client with a unique subdomain of ir.md.
I am the Founder & Executive Director of ir.md, and handle operations management.
Organization Website: https://nic.ir.md/

Reason for PSL Inclusion
====

We provide each hosting client with a unique subdomain of ir.md.

We need our customers' applications and websites to be isolated from each other, in relation to cookies, etc.

Customers with growth potentially incoming.

Number of users this request is being made to serve: +1K

DNS Verification via dig
=======
```
dig +short TXT _psl.ir.md
"https://github.com/publicsuffix/list/pull/1625"
```

Results of Syntax Checker (`make test`)
=========

### CODE REVIEW

**Comment by irancms** [comment]
up

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12565,6 +12565,10 @@ iopsys.se
 // Submitted by Matthew Hardeman <mhardeman@ipifony.com>
 ipifony.net
 
+// ir.md : https://nic.ir.md
+// Submitted by Ali Soizi <ali.soizi@gmail.com>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:12569]: If possible, please use a non-personal address.

**Review by simon-friedberger — CHANGES_REQUESTED** [review]

**Comment by simon-friedberger** [comment]
* [x] ir.md expires 2028-09-07
* [ ] DNS _psl entries
* [x] Tests pass
* [x] Sorting according to guidelines
* [ ] Reasoning/Organization description

**Comment by simon-friedberger** [comment]
Organization website unreachable so I assume this is now obsolete.

**Comment by irancms** [comment]
> Organization website unreachable so I assume this is now obsolete.

Hello
A pre-scheduled update has been completed and the website is now available.

**Comment by simon-friedberger** [comment]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [x] ir.md expires 2028-09-07
* [x] DNS _psl entries
* [x] Tests pass
* [x] Sorting
* [x] Reasoning/Organization description

**Comment by dnsguru** [comment]
it appears that simon has approved this.   Merging

**Comment by groundcat** [comment]
Hello @irancms , it seems that the project or organization associated with your domain, which was merged into the Public Suffix List (PSL), may no longer be active based on the observable usage of subdomains. 

- `ir.md` TIMEOUT (see #2172)

If the service has been decommissioned and the domain is no longer linked to an active project, as outlined in your original request, please confirm whether it can be safely removed from the PSL. This will help prevent leaving unnecessary entries for PSL volunteers and future registrants, as requesters often "set and forget" and neglect to request the removal of their domains from the PSL when their project reaches its end of life (EOL).

The PSL is a globally used resource, and it's important to understand that tens of millions, if not hundreds of millions, of devices and users incorporate the list. We aim to keep it as minimal as possible for those who truly need PSL inclusion. Projects that are smaller in scale, such as those without thousands of users or subdomains belonging to thousands of individual entities, will likely be ineligible for inclusion. If you no longer need your entry to remain on the PSL, **please create a new pull request to remove it** or reply here to confirm.

If your project or organization is still active and requires continued inclusion in the PSL, please reply at your earliest convenience. If we do not hear back from you for an extended period, **your entry may be subject to removal**. 

Additionally, ensure that the required `_psl` TXT record is maintained, as specified in the [PSL Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines#validation-of-submissions--always), and that your domain is kept renewed with an expiry date of more than 2 years at all times to avoid future removal.

Thank you.

**Comment by irancms** [comment]
Hello,

First, I sincerely apologize for the issue and any confusion caused. We were in the process of a server migration, which led to the temporary downtime of our domain and subdomains. I’m pleased to inform you that the domain is now functioning correctly, and the project remains active.

We appreciate your efforts in maintaining the Public Suffix List and understand the importance of keeping it up to date. Please rest assured that the necessary _psl TXT record is being maintained, and the domain renewal is in good standing with more than two years before expiration.

Thank you for your understanding, and I look forward to ensuring continued inclusion in the PSL.

Best regards,
Ali Soizi

**Comment by irancms** [comment]
> Hello @irancms , it seems that the project or organization associated with your domain, which was merged into the Public Suffix List (PSL), may no longer be active based on the observable usage of subdomains.
> 
> * `ir.md` TIMEOUT (see [Candidates for removal #2172](https://github.com/publicsuffix/list/issues/2172))
> 
> If the service has been decommissioned and the domain is no longer linked to an active project, as outlined in your original request, please confirm whether it can be safely removed from the PSL. This will help prevent leaving unnecessary entries for PSL volunteers and future registrants, as requesters often "set and forget" and neglect to request the removal of their domains from the PSL when their project reaches its end of life (EOL).
> 
> The PSL is a globally used resource, and it's important to understand that tens of millions, if not hundreds of millions, of devices and users incorporate the list. We aim to keep it as minimal as possible for those who truly need PSL inclusion. Projects that are smaller in scale, such as those without thousands of users or subdomains belonging to thousands of individual entities, will likely be ineligible for inclusion. If you no longer need your entry to remain on the PSL, **please create a new pull request to remove it** or reply here to confirm.
> 
> If your project or organization is still active and requires continued inclusion in the PSL, please reply at your earliest convenience. If we do not hear back from you for an extended period, **your entry may be subject to removal**.
> 
> Additionally, ensure that the required `_psl` TXT record is maintained, as specified in the [PSL Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines#validation-of-submissions--always), and that your domain is kept renewed with an expiry date of more than 2 years at all times to avoid future removal.
> 
> Thank you.

Hello,

First, I sincerely apologize for the issue and any confusion caused. We were in the process of a server migration, which led to the temporary downtime of our domain and subdomains. I’m pleased to inform you that the domain is now functioning correctly, and the project remains active.

We appreciate your efforts in maintaining the Public Suffix List and understand the importance of keeping it up to date. Please rest assured that the necessary _psl TXT record is being maintained, and the domain renewal is in good standing with more than two years before expiration.

Thank you for your understanding, and I look forward to ensuring continued inclusion in the PSL.

Best regards,
Ali Soizi

**Comment by groundcat** [comment]
Hi @irancms the organization website https://nic.ir.md/ is not working. Any updates? Based on https://www.google.com/search?q=site%3Air.md it appears that the project or organization associated with your domain, which was previously added to the Public Suffix List (PSL), may no longer be active based on current domain usage and availability checks. If the service has been discontinued and the domain is no longer associated with an active project, please confirm whether the entry can be safely removed from the PSL. _This helps keep the list accurate and avoids retaining unused entries when projects reach end of life. The PSL is a widely used resource relied upon by many applications and users worldwide, so entries should remain limited to projects that continue to require inclusion._ If your project is still active and requires continued PSL coverage, **please reply with confirmation** and ensure that the required verification records are maintained according to the [PSL guidelines](https://github.com/publicsuffix/list/wiki/Guidelines), and that the domain remains properly renewed. If no response is received after a reasonable period, the entry may be considered for removal. Thank you.

**Pull request merged**